sdk/core/ts-http-runtime/review/azure-core-comparison.diff

```diff
@@ -572,7 +572,35 @@ index 609878a..da6742a 100644
     *
     * type MyClient = Client & {
     *   path: Routes;
-@@ -333,11 +314,9 @@ export type ClientOptions = PipelineOptions & {
+@@ -217,18 +198,6 @@ export interface Client {
+   pathUnchecked: PathUnchecked;
+ }
+ 
+-/**
+- * A Node.js Readable stream that also has a `destroy` method.
+- */
+-export interface NodeJSReadableStream extends NodeJS.ReadableStream {
+-  /**
+-   * Destroy the stream. Optionally emit an 'error' event, and emit a
+-   * 'close' event (unless emitClose is set to false). After this call,
+-   * internal resources will be released.
+-   */
+-  destroy(error?: Error): void;
+-}
+-
+ /**
+  * Http Response which body is a NodeJS stream object
+  */
+@@ -236,7 +205,7 @@ export type HttpNodeStreamResponse = HttpResponse & {
+   /**
+    * Streamable body
+    */
+-  body?: NodeJSReadableStream;
```

Comment:
Can you open an issue for how we want to address this in Unbranded?

I think we will want to do something more broad and include this change in the pipeline as well but not totally sure what it would look like yet

Reply:
I opened https://github.com/Azure/azure-sdk-for-js/issues/33436